File: codes/java/Code_2_03_1.java
import java.util.*;

class Code_2_03_1 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.println(func1()); // 「2021」と出力
		System.out.println(func2(500)); // 「1501」と出力
		System.out.println(func2(500)); // 「1502」と出力
	}
	static int cnt = 1000;
	static int func1() {
		return 2021;
	}
	static int func2(int pos) {
		cnt += 1;
		return cnt + pos;
	}
};

File: codes/python/Code_2_03_1.py
cnt = 1000

def func1():
	return 2021

def func2(pos):
	global cnt # グローバル変数 cnt を関数の内部で使う
	cnt += 1
	return cnt + pos

print(func1()) # 「2021」と出力
print(func2(500)) # 「1501」と出力
print(func2(500)) # 「1502」と出力
